pam/gdm/conversation_test.go

```diff
@@ -99,3 +129,569 @@ func TestSendToGdm(t *testing.T) {
 		})
 	}
 }
+
+func TestDataSend(t *testing.T) {
+	t.Parallel()
+	t.Cleanup(pam_test.MaybeDoLeakCheck)
+	testCases := map[string]struct {
```

Comment:
Here we should have a whitespace between actually defining the cases and the test markings